### PR TITLE
Fix form and update css for more fluid transition on homepage

### PR DESCRIPTION
**Pull request by alexkaufman06**
I deleted the "require background" from the application.js file because it was breaking the app when I ran it on my computer.  Some changes where made to the schema.rb file as well after I created the database.  I don't think any of these changes will be harmful but it's worth checking before pushing to production.  Other than than, I just made some styling changes and fixed the form spacing.

### CODE REVIEW

**Comment by davidmolina** [comment]
Thanks @alexkaufman06. Prior to this, the top container covers the second container. Shouldn't cover 2nd container I don't believe. cc/ @nanoxd

File: db/schema.rb [hunk 1]
```diff
@@ -13,6 +13,9 @@
 
 ActiveRecord::Schema.define(version: 20141202222312) do
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
```
Comment (nanoxd) [inline db/schema.rb]: Didn't know about this change. Is it necessary to make the form work?

Comment (davidmolina) [inline db/schema.rb]: Not sure. Form for petition?

Comment (alexkaufman06) [inline db/schema.rb]: I think this change got added by PostgreSQL when I created the database on my computer.

**Comment by alexkaufman06** [comment]
@davidmolina what container are you referring to?

**Comment by davidmolina** [comment]
The top one w/ the b-roll.

**Comment by davidmolina** [comment]
@nanoxd - about to accept these changes. Ok?

**Pull request merged**